action/container/ssh_test.go

```diff
@@ -101,6 +102,13 @@ func TestOpenSSH(t *testing.T) {
 			myContainer, err := Setup(ctx, client, &tc.opts, "")
 			assert.NoError(t, err)
 
+			// Set timeout for opening the SSH server
+			timeout := make(chan bool, 5)
+			go func() {
```

Comment:
The more idiomatic approach is to create a 5 second ticker and check if a tick occurred in the select statement. Afterwards stop the ticker. Saves you the manual goroutine with a timeout channel.